airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -260,6 +284,11 @@ def _update_main_container_yaml(self, container: dict):
             container["image"] = self.image
         if self.image_pull_policy:
             container["imagePullPolicy"] = self.image_pull_policy
+        if self.volume_files_from_secret:
```

Comment:
Problem here that this will be updated just to the main container. Hum.. this is ok but has caviats. Please update the help to reflect this. If should clearly state that this updates the main container only.